sqlglot/parser.py

```diff
@@ -326,7 +332,6 @@ class Parser(metaclass=_Parser):
     FACTOR = {
         TokenType.DIV: exp.IntDiv,
         TokenType.LR_ARROW: exp.Distance,
-        TokenType.SLASH: exp.Div,
```

Comment:
can we leave this for completeness and only override it if it's floatdiv?

Reply:
sure